boards/waveshare/esp32s3_touch_lcd_1_28/esp32s3_touch_lcd_1_28_esp32s3_appcpu.dts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2024 Joel Guittet
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <espressif/esp32s3/esp32s3_appcpu.dtsi>
+
+/ {
+	model = "ESP32-S3-Touch-LCD-1.28 APPCPU";
+	compatible = "waveshare,esp32-s3-touch-lcd-1.28";
+
+	chosen {
+		zephyr,sram = &sram0;
+	};
+};
+
+&cpu0 {
+	clock-frequency = <ESP32_CLK_CPU_240M>;
+};
+
+&cpu1 {
+	clock-frequency = <ESP32_CLK_CPU_240M>;
+};
+
+&flash0 {
+	status = "okay";
+	reg = <0x0 DT_SIZE_M(16)>;
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x0000F000>;
+			read-only;
+		};
+
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x00010000 0x00100000>;
+		};
+
+		slot1_partition: partition@110000 {
+			label = "image-1";
+			reg = <0x00110000 0x00100000>;
+		};
+
+		scratch_partition: partition@210000 {
+			label = "image-scratch";
+			reg = <0x00210000 0x00040000>;
+		};
```

Comment:
Would remove and give to slots instead since swap using scratch is not recommended

Reply:
Agree but it's retrieved from the esp32s3_devkitm board from Espressif. I guess it's better to keep consistency across similar boards?

Reply:
@nordicjm what is your opinion?

Reply:
@sylvioalves

Reply:
@sylvioalves ? Your opinion about this ? What should be the default in zephyr for this kind of board ?

Reply:
@nordicjm hey since we are on the topic I'll add a question: do you think it would make sense to have few templates for in-tree board partitions maps? We have a lot of boards based on esp32, nrf52840 etc... and the flash partitioning is pretty much always the same, maybe we can have some dtsi and `#include` it in the board file? As much as I dislike mid-structure includes, it may be better than repeating this to infinity.